src/v/cluster/errors.cc

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "cluster/errc.h"
+
+#include <iostream>
+
+namespace cluster {
+std::ostream& operator<<(std::ostream& o, cluster::errc err) {
```

Comment:
could we just use `errc_category::message()`?

Reply:
I saw that, that was formatted as a message "this is a sequence violation.." or some such which doesn't format well (grammatically) with our log lines, so used an explicit formatter.